routers/router.php

```diff
@@ -146,6 +147,8 @@ protected function get_routing_table_name($ip_version = 'ipv6') {
 
   protected abstract function build_aspath_regexp($parameter);
 
+  abstract protected function build_aspath_regexp6($parameter);
```

Comment:
Not required if the IPv6 specific AS path regexp function is removed and its logic included in the existing AS path function.